src/api/app/controllers/application_controller.rb

```diff
@@ -460,14 +460,14 @@ def shutup_rails
   end
 
   def set_nobody
-    User.current = User.find_nobody!
+    User.set_current(User.find_nobody!)
   end
 
   def set_influxdb_data
-    anonymous = User.current_login == User::NOBODY_LOGIN
+    anonymous = User.possibly_nobody.is_nobody?
```

Comment:
```anonymous?``` would be a better name for a boolean variable, no?

Reply:
out of scope

Reply:
When I see a `foo?` somewhere I would expect it to be a method. I guess others would have the same "issue".

Anyway, it's not possible in Ruby to have `?` in variable names.

Reply:
@bgeuken a method fits well here.